### PR TITLE
MeasureTrackQuality

### PR DESCRIPTION
**Pull request by DanRuderman**
This should be pretty complete. Has unit tests now.

### CODE REVIEW

**Comment by DanRuderman** [comment]
@mcquin @0x00b1 

We'll need to update the book chapter with instructions on
downloading plugins. Can you provide me the URL we should
include that provides the instructions?

Is everything still be on track for CPv3 and the plugin to be released
by the end of September (when we expect the book to be published)?

**Comment by mcquin** [comment]
> We'll need to update the book chapter with instructions on
downloading plugins. Can you provide me the URL we should
include that provides the instructions?

We'll keep up-to-date instructions for installing plugins in the [README](https://github.com/CellProfiler/CellProfiler-plugins/blob/master/README.md) at the root of this repository. Additionally, and if you prefer, we can create a page in the repository wiki on setting up plugins.

> Is everything still be on track for CPv3 and the plugin to be released
by the end of September (when we expect the book to be published)?

I think it's safe to say "yes" to CPv3 being out by the end of September. This module will be available well before then. :)

File: measuretrackquality.py [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+<p>
+<h4>Publications</h4>
+Patsch, K <i>et al.</i>, <a href=https://www.nature.com/articles/srep34785>Single cell
+dynamic phenotyping</a>, <i>Scientific Reports</i> 6:34785 (2016)
+<p>
+<h4>Author</h4>
+Dan Ruderman (ruderman@usc.edu)
+""".format(**{"min_tp":TRAM_MIN_TIME_POINTS})
+
+# Features to add:
```
Comment (mcquin) [inline measuretrackquality.py]: @DanRuderman I'm curious if you're planning on implementing these before this module gets merged into master?

**Comment by DanRuderman** [comment]
> We'll keep up-to-date instructions for installing plugins in the README at the root of this repository. Additionally, and if you prefer, we can create a page in the repository wiki on setting up plugins.

I'm wondering what the easiest route would be for installation. Would we expect all CP users to have git installed already? The instructions in the README assume that you do. Would there instead be a way to directly download the current repository as a zip file instead of cloning?

File: measuretrackquality.py (hunk 1 above)
Comment (DanRuderman) [inline measuretrackquality.py]: Probably not. More of a note to myself (and others) about what the next steps could be. I'd like to see how people use it before prioritizing. I can remove those lines if they are out of place.

**Comment by mcquin** [comment]
Hi @DanRuderman,

I apologize for the long delay in communication. It has been a very busy month. 

> Would there instead be a way to directly download the current repository as a zip file instead of cloning?
Sure! Rather than using git to clone the repository, users can download the repository [using this link](https://github.com/CellProfiler/CellProfiler-plugins/archive/master.zip) which is also available from the "Clone or download" button on the front page of the repository.

**Comment by DanRuderman** [comment]
@mcquin Hi Claire. Just checking to see if there is any further estimates on release dates (for plugins and CPv3). Thanks!

**Comment by mcquin** [comment]
Hi @DanRuderman Thanks for checking in! We're planning on drafting a release candidates for CP 3 on 4 September.

**Comment by DanRuderman** [comment]
@mcquin Great. Best of luck with the roll-out!

**Comment by DanRuderman** [comment]
@mcquin Congrats on v3's release!  Any time frame for merging in this pull request?

**Comment by mcquin** [comment]
Thanks @DanRuderman! I think we can merge this if you're happy with its current functionality. Do you have a pipeline available that you/we could use to verify compatibility with 3.0?

**Comment by DanRuderman** [comment]
@mcquin I just did a quick test and it seems to be working. But would be great if you could try it out separately. Dropbox link to a project and input images is [here](https://www.dropbox.com/sh/4vbfmih11dcmwzh/AABCNCEmvT0ITybz_Z7-A_i1a?dl=0). You'll need to update the output and plugin directories, etc, within the project. Hopefully will be straightforward and run OK for you. Please let me know if I can be of any assistance!

**Comment by mcquin** [comment]
Awesome, @DanRuderman. This works really well. :)

Thanks for working with us on this contribution!

**Comment by DanRuderman** [comment]
@mcquin Yay!!

**Pull request merged**